docs/content/guides/dagster/example_project.mdx

```diff
@@ -75,8 +75,9 @@ The way we model resources helps separate the business logic in code from enviro
 
 [I/O managers](/concepts/io-management/io-managers) - An I/O manager is a special kind of resource that handles storing and loading assets. This example includes [a wide range of I/O managers](https://github.com/dagster-io/dagster/tree/master/examples/project_fully_featured/project_fully_featured/resources) such as:
 
-- [`DuckDBPartitionedParquetIOManager`](https://github.com/dagster-io/dagster/blob/master/examples/project_fully_featured/project_fully_featured/resources/duckdb_parquet_io_manager.py): interacts with Spark and dbt without any long-running process. It minimizes setup difficulty and is useful for local development.
-- [`SnowflakeIOManager`](https://github.com/dagster-io/dagster/blob/master/examples/project_fully_featured/project_fully_featured/resources/snowflake_io_manager.py): handles outputs that are either Spark or Pandas DataFrames and write data to a Snowflake table specified by metadata on the relevant <PyObject object="Out" />. The metadata is helpful for observability, especially in production.
+- [`ParquetIOManager`](https://github.com/dagster-io/dagster/blob/master/examples/project_fully_featured/project_fully_featured/resources/parquet_io_manager.py): stores outputs as parquet files in your local file system. It minimizes setup difficulty and is useful for local development.
+- [DuckDBIOManager](/\_apidocs/libraries/dagster-duckdb#dagster_duckdb): Dagster provided DuckDB I/O manager that can store and load data as Pandas or PySpark DataFrames. Useful for local development since DuckDB runs locally and requires minimal setup.
+- [`SnowflakeIOManager`](/integrations/snowflake): Dagster provided Snowflake I/O manager that can store and load data as Pandas or PySpark DataFrames. 
```

Comment:
nit: maybe link use the `PyObject` for "DuckDBIOManager" and "SnowflakeIOManager", so it shows up like a `code` which could sort of differentiate the natively supported io managers from the custom ones.

Reply:
should we also mention partition support here?

Reply:
> re using PyObject

I like that idea, but i'm worried it could be confusing because the object we would link to is the `build_duckdb_io_manager` and `build_snowflake_io_manager` functions, not something called DuckDBIOManager or SnowflakeIOManager - what do you think?